### PR TITLE
Issue#786

### PR DESCRIPTION
**Pull request by nfukuzaki**

### CODE REVIEW

File: ThingIFSDK/ThingIFSDK/Command.swift [hunk 1]
```diff
@@ -163,17 +189,29 @@ public class Command: NSObject, NSCoding {
                 command = Command(commandID: commandID, targetID: targetID!, issuerID: issuerID!, schemaName: schemaName!, schemaVersion: schemaVersion!, actions: actionsArray, actionResults: actionsResultArray, commandState: commandState)
         }
         if command != nil {
+            let firedByTriggerID = nsDict["firedByTriggerID"] as? String
```
Comment (yongpingchen) [inline ThingIFSDK/ThingIFSDK/Command.swift]: can be improve by 

``` swift
command?.firedByTriggerID = nsDict["firedByTriggerID"] as? String
```

File: ThingIFSDK/ThingIFSDK/Command.swift [hunk 2]
```diff
@@ -163,17 +189,29 @@ public class Command: NSObject, NSCoding {
...
+            if firedByTriggerID != nil {
+                command!.firedByTriggerID = firedByTriggerID
+            }
+            let createdAt = nsDict["createdAt"] as? NSNumber
```
Comment (yongpingchen) [inline ThingIFSDK/ThingIFSDK/Command.swift]: can be improved by 

``` swift
        if let createdAt = nsDict["createdAt"] as? NSNumber {
            command?.created = NSDate(timeIntervalSince1970: (createdAt.doubleValue)/1000.0)
        }
```

File: ThingIFSDK/ThingIFSDK/Command.swift [hunk 3]
```diff
@@ -163,17 +189,29 @@ public class Command: NSObject, NSCoding {
...
+            if createdAt != nil {
+                command!.created = NSDate(timeIntervalSince1970: (createdAt!.doubleValue)/1000.0)
+            }
+            let modifiedAt = nsDict["modifiedAt"] as? NSNumber
```
Comment (yongpingchen) [inline ThingIFSDK/ThingIFSDK/Command.swift]: can be improve same as createdAt

**Comment by nfukuzaki** [comment]
@yongpingchen Fixed!!

**Comment by yongpingchen** [comment]
LGTM

**Pull request merged**